pkg/services/ngalert/state/historian/loki.go

```diff
@@ -312,6 +321,7 @@ func StatesToStream(rule history_model.RuleMeta, states []state.StateTransition,
 		samples = append(samples, Sample{
 			T: state.State.LastEvaluationTime,
 			V: line,
+			M: removePrivateLabels(state.Annotations),
```

Comment:
In my experience, most alerts do not have annotations that are suitable for use in structured metadata - I personally think the alert labels would be a better candidate for it instead - but YMMV!